dss/storage/index_document.py

```diff
@@ -476,6 +477,7 @@ class BundleTombstoneDocument(IndexDocument):
     def from_replica(cls, replica: Replica, tombstone_id: TombstoneID, logger):
         blobstore, _, bucket_name = Config.get_cloud_specific_handles(replica)
         tombstone_data = json.loads(blobstore.get(bucket_name, tombstone_id.to_key()))
+        tombstone_data['uuid'] = tombstone_id.uuid
```

Comment:
I would

```
self = cls(…)
self['uuid'] = …
```

to keep it symmetric with BundleDocument.

Reply:
Nice suggestion. Done.